drivers/usb/chipidea/ci_hdrc_imx.c

```diff
@@ -62,7 +62,6 @@ static const struct ci_hdrc_imx_platform_flag imx6ul_usb_data = {
 };
 
 static const struct ci_hdrc_imx_platform_flag imx7d_usb_data = {
-	.flags = CI_HDRC_SUPPORTS_RUNTIME_PM,
```

Comment:
What is the content of flags, then? It's not initialized if you just remove this line.

Reply:
As discussed with @zandrey, I'll change this implementation with the one from NXP